playbooks/roles/deploy-osh/templates/keystone.yaml.j2

```diff
@@ -1,2 +1,42 @@
 ---
-{{ lookup('template','files/common-images-overrides.yml') | from_yaml | to_nice_yaml(indent=2) }}
+{% if developer_mode %}
+conf:
+  apache2:
+    binary: apache2ctl
+    start_parameters: -DFOREGROUND -k start
+    site_dir: /etc/apache2/vhosts.d
+    conf_dir: /etc/apache2/conf.d
+    a2enmod:
+      - version
+    security: |
+      <Directory "/var/www">
+      Options Indexes FollowSymLinks
+      AllowOverride All
+      <IfModule !mod_access_compat.c>
+        Require all granted
+      </IfModule>
+      <IfModule mod_access_compat.c>
+        Order allow,deny
+        Allow from all
+      </IfModule>
+      </Directory>
+pod:
+  security_context:
+    keystone:
+      pod:
+        runAsUser: 0
```

Comment:
I think we also need
```
      container:
        keystone_api:
          readOnlyRootFilesystem: false
```

So that apache is able to write into `/etc/apache2/sysconfig.d/`

Reply:
It is set to false by default with https://review.openstack.org/#/c/644627/

Reply:
Valid point, could you update?

Reply:
updated!